packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -74,6 +74,9 @@ foreman-installer --scenario foreman --migrations-only > /dev/null
 %{_mandir}/man8
 
 %changelog
+* Tue Jun 19 2018 Ondrej Prazak <oprazak@redhat.com> - 1.18.0-0.1.RC2
+- Release 1.18.0-RC1
```

Comment:
Should be RC2 I guess. Could you also add a changelog for RC1 for completeness? Looks like this applies to all specs.